src/main/scala/com/ibm/couchdb/api/Documents.scala

```diff
@@ -132,7 +163,11 @@ class Documents(client: Client, db: String, typeMapping: TypeMapping) {
       Res.Error("cannot_delete", "The attachment name is empty").toTask[Res.DocOk]
     else
       client.delete[Res.DocOk](
-        s"/$db/${ obj._id }/$name?rev=${ obj._rev }",
+        s"/$db/${
+          obj._id
+        }/$name?rev=${
+          obj._rev
+        }",
```

Comment:
Is this change and a few above a result of auto-formatting? It seems to me that string literals are easier to read when they are on a single line.